kpatch-build/create-diff-object.c

```diff
@@ -205,6 +205,15 @@ static void kpatch_bundle_symbols(struct kpatch_elf *kelf)
 	}
 }
 
+static void kpatch_add_child_symbol(struct symbol *parent, struct symbol *child)
+{
+	parent->nb_children++;
+	parent->children = realloc(parent->children,
+				   sizeof(*parent->children) * parent->nb_children);
```

Comment:
A nit, but realloc() could fail so we should catch that like other allocation failures.

Reply:
Yes, will update.

Reply:
It also now needs to be freed at some point.